src/features/profile/customerBerths/orderInfo/OrderStatusLabel.tsx

```diff
@@ -0,0 +1,50 @@
+import { StatusLabel, StatusLabelProps } from 'hds-react';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { OrderStatus } from '../../../../__generated__/globalTypes';
+
+export interface OrderStatusProps {
+  status: OrderStatus;
+}
+
+const OrderStatusLabel = ({ status }: OrderStatusProps) => {
+  const { t } = useTranslation();
+  const getStatusLabelKey = (status: OrderStatus) => {
+    switch (status) {
+      case OrderStatus.CANCELLED:
+        return 'common.cancelled';
+      case OrderStatus.ERROR:
+        return 'common.error';
+      case OrderStatus.EXPIRED:
+        return 'common.expired';
+      case OrderStatus.PAID:
+        return 'common.paid';
+      case OrderStatus.REJECTED:
+        return 'common.rejected';
+      case OrderStatus.WAITING:
+        return 'common.waitingForPayment';
```

Comment:
I think we should fall back to whatever comes from the backend in case they add new statuses?
```suggestion
        return 'common.waitingForPayment';
     default:
        return status;
```

And to take it one step further, a unit test that catches a missing translation might be useful, as we do on the admin UI.